NDTensors/src/lib/Sectors/src/abstractcategory.jl

```diff
@@ -1,45 +1,129 @@
+# This file defines the abstract type AbstractCategory
+# all fusion categories (Z{2}, SU2, Ising...) are subtypes of AbstractCategory
+
 abstract type AbstractCategory end
 
-label(c::AbstractCategory) = error("method `label` not defined for type $(typeof(c))")
+# ===================================  Base interface  =====================================
+function Base.isless(c1::C, c2::C) where {C<:AbstractCategory}
+  return isless(category_label(c1), category_label(c2))
+end
 
-function dimension(c::AbstractCategory)
-  return error("method `dimension` not defined for type $(typeof(c))")
+# =================================  Sectors interface  ====================================
+trivial(x) = trivial(typeof(x))
+function trivial(axis_type::Type{<:AbstractUnitRange})
+  return GradedAxes.gradedrange([trivial(eltype(axis_type))])  # always returns nondual
+end
+function trivial(la_type::Type{<:LabelledNumbers.LabelledInteger})
+  return la_type(1, trivial(LabelledNumbers.label_type(la_type)))
```

Comment:
A bit more generic (say to preserve the integer type, account for the type potentially changing, etc.):
```suggestion
  return LabelledNumbers.labelled(one(LabelledNumbers.label_type(la_type)), trivial(LabelledNumbers.label_type(la_type)))
```